rfd/0066-ip-based-validation.md

```diff
@@ -0,0 +1,85 @@
+---
+authors: Przemko Robakowski(przemko.robakowski@goteleport.com)
+state: draft
+---
+
+# RFD 66 - IP-based validation
+
+# Required Approvers
+
+* Engineering @zmb3 && (@codingllama || @nklaassen )
+* Product: (@xinding33 || @klizhentas )
+
+## What
+
+Additional validation based on client IP for creating and using certificates. User can define which IP addresses can
```

Comment:
With the latest changes, users don't define which IP addresses can create certificates, right?

Reply:
Yes, I've updated it